Dockerfile

```diff
@@ -43,18 +53,18 @@ COPY observiq-otel-collector /collector/observiq-otel-collector
 COPY --from=jmxjar /opentelemetry-java-contrib-jmx-metrics.jar /opt/opentelemetry-java-contrib-jmx-metrics.jar
 COPY plugins /etc/otel/plugins
 
-RUN adduser \
-    --disabled-password \
-    --gecos "" \
-    --no-create-home \
-    --uid 10005 \
-    otel
-
 RUN echo "output: stdout\nlevel: info\n" > /etc/otel/logging.yaml
 ENV LOGGING_YAML_PATH=/etc/otel/logging.yaml
 
+# Default config allows the collector to run without an injected config, which is required
+# when connecting to an OpAMP platform.
+RUN echo "receivers:\n  hostmetrics:\n    collection_interval: 1h\n    scrapers:\n      load:\nexporters:\n  logging:\nservice:\n  pipelines:\n    metrics:\n      receivers: [hostmetrics]\n      exporters: [logging]" > /etc/otel/config.yaml
```

Comment:
Any reason we don't copy the config from the repo when this is built? I guess do we plan on people building the dockerfile themselves or just operating on pre-built images?